src/scss/objects/_form-toggle.scss

```diff
@@ -11,10 +11,12 @@ $formToggle-border-color-checked    : $grey--lighter;
 $formToggle-border-width            : 1px;
 $formToggle-border-radius           : 8px;
 $formToggle-tick-background         : $green;
+$formToggle-tick-hover-background   : $grey--lighter;
```

Comment:
Wonder if this (and the variable above) should be `formToggle-icon-hover-background` so that if the icon changes in future the variable name doesn't need to be updated.